examples/deck.gl/3d-tiles/app.js

```diff
@@ -256,7 +257,8 @@ export default class App extends PureComponent {
       onTilesetLoad: this._onTilesetLoad,
       onTileLoad: this._onTilesetChange,
       onTileUnload: this._onTilesetChange,
-      onTileLoadFail: this._onTilesetChange
+      onTileLoadFail: this._onTilesetChange,
+      loadOptions
```

Comment:
@xintongxia Seems a bit strange to call this `loadOptions. `tilesetOptions` would be more appropriate. I will make a note in our tracker issue